spec/models/like_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  describe 'associations' do
+    it { should belong_to(:user).with_foreign_key(:author_id) }
+    it { should belong_to(:post) }
+  end
+
+  describe 'callbacks' do
+    context 'after_save' do
+      let(:user) { FactoryBot.create(:user) }
+      let(:post) { FactoryBot.create(:post) }
+    end
+  end
```

Comment:
- [ ] As required for the project any custom methods in the models to have a corresponding unit test, Kindly ensure that you add unit test(s) to `#update_likes_counter` method in the `Like` class.